src/runtime/EnumerateObject.cpp

```diff
@@ -201,6 +201,7 @@ void EnumerateObjectWithIteration::executeEnumeration(ExecutionState& state, Enc
     m_hiddenClassChain.push_back(m_object->structure());
 
     std::unordered_set<String*, std::hash<String*>, std::equal_to<String*>, GCUtil::gc_malloc_allocator<String*>> keyStringSet;
+    std::unordered_set<Object*> protoChainSet;
```

Comment:
It seems that this `protoChainSet` is defined but not used at all

Reply:
Oops